src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -2408,7 +2465,7 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
                                                             value={this.utilitiesMenuSelection.selectedGeneOption ? this.utilitiesMenuSelection.selectedGeneOption.value : undefined}
                                                             onChange={this.onUtilitiesGeneSelect}
                                                             isLoading={this.horzGeneOptions.isPending}
-                                                            options={this.utilityMenuGeneOptions}
+                                                            options={this.utilityMenuGeneOptions.result}
```

Comment:
looks like `this.utilityMenuGeneOptions` is now a mobxpromise, so it should be inserted into the `promises` array on line 2266 to be awaited

Reply:
The result of `utilityMenuGeneOptions` is only expected in a specific case, and I don't think it should affect the display of the entire plot. When the promise is pending, the ReactSelect component that shows the gene options will be in a loading state and I think that might be enough.

Reply:
okay that makes sense to me.